components/FrontPage/Combined/WriteOnceAccessAnywhere.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import TBox from "../textbox";
+import Btn from "../button";
+import Image from "next/image";
+
+// This is the 2nd to last section with the "write, once access anywhere" catchphrase
+
+export default function WriteOnceAccessAnywhere() {
+  return (
+    <div className="flex h-[700px] w-screen justify-center bg-slate-200 dark:bg-slate-800">
+      <div className="flex h-full max-w-[1200px] items-center justify-center gap-24 px-6">
+        <div className="ml-3 max-w-[90%] sm:max-w-[50%]">
```

Comment:
Instead of 90% or 50%, prefer tailwind presets: 1/2 or 10/12.

Reply:
Impossible, tailwind does not have such presets for max-w, only for regular width.

Reply:
max-w-[50%] for now then